mantle/kola/tests/ignition/journaldEntry.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"strconv"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.ignition.journald-log",
+		Run:         sendJournaldLog,
+		ClusterSize: 1,
+		UserDataV3: conf.Ignition(`{
+    "ignition": {"version": "3.0.0"},
+    "systemd": {
+        "units": [{
+			"enabled": false,
+			"name": "zincati.service"
```

Comment:
Agreed. Ignition should execute anyway as Colin noted. If you want to be extra explicit then it might be clearer to use ignition to write an empty file and comment above the register line that the purpose is to be sure Ignition executes and noting that it should anyway, but this is a fallback "just in case". Though again, I don't think it's really required.